### ISSUE TITLE
Excel Function Arguments popup displaying #value for string parameter with custom function 

### ISSUE THREAD

**Issue by vivek-insight** [OfficeDev/office-js#4853]
We have created custom functions for excel. When we add a new function using the Excel Function Arguments window (fx) it works fine. But after saving the sheet and opening the sheet again, after that when we open function arguments for the same formula then #value is displayed for all string parameters, and all other parameters displays number as there type. Same issue also occurs if we copy formula from one sheet to another and open it in second sheet. 
I am currently using manifest file reference from shared folder.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC desktop
* Host [Excel, Word, PowerPoint, etc.]: Excel
* Office version number: Microsoft® Excel® for Microsoft 365 MSO (Version 2408 Build 16.0.17928.20114) 64-bit
* Operating System: Window 11
* Browser (if using Office on the web): N/A
 
## Expected behavior
The Excel Function Arguments window should correctly show the values for each parameter.

## Current behavior
After saving when we again open the excel and select the formula, the value of string parameters are shown as #value in the Excel Function Arguments window

## Steps to reproduce

1. Create and build the yo office custom function with reactjs.
2. Load the manifest using the instructions to Sideload Office Add-ins for testing from a network share.
3. Open the excel function arguments windows and add newly created formula.
4. Save the excel and open the excel again
5. After opening the excel again click the already added formula and open function arguments window now you can see string parameter displaying #value 
6. you can also try copying of formula from one sheet to another and open formula on second sheet same issue occurs

## Context
Users will be confused and consider this an error with the Add-in custom function

Before Save
![before_Save](https://github.com/user-attachments/assets/41c89593-7295-4ccc-ae53-b51849f40f17)

After save / or copy paste
![After_Save](https://github.com/user-attachments/assets/af011f4b-9055-41e1-bb50-02311ad22af3)

**Comment by shanshanzheng-dev** [comment]
Hi @vivek-insight Thanks for reporting this issue. We'll take a look and report back if we have a suggestion for you.

**Comment by vivek-insight** [comment]
@shanshanzheng-dev Hi did you find any solution ?

**Comment by shanshanzheng-dev** [comment]
Hi @vivek-insight Sorry for slow response. This is a our new feature. All codes are checked in. We will update for you once the feature is fully release to production. It's around several weeks after.

**Comment by vivek-insight** [comment]
@shanshanzheng-dev any tentative date when we can expect this feature.

**Comment by qinliuMSFT** [comment]
Hi @vivek-insight, the feature should be available on the current monthly channel (Version 2409 Build 18025.xxxxx and later) on Win32 platform. Please let me know if you are able to use it.

**Comment by vivek-insight** [comment]
@qinliuMSFT issue is still replicable. currently using **Microsoft® Excel® for Microsoft 365 MSO (Version 2409 Build 16.0.18025.20160) 64-bit**  excel version. Any tentative date when we can expect the same in 64 bit platform

**Comment by vivek-insight** [comment]
@XuanZhouMSFT @shanshanzheng-dev  Any update when we can expect the fix

**Comment by sunyao-ms** [comment]
Hi, @vivek-insight ，Sorry to hear that. Can you help confirm when you encountered the issue, were there any other Excel workbooks open? If so, was the current workbook not the first workbook? If the answer is yes, this is a known issue that we are currently working to fix.
cc: @XuanZhouMSFT

**Comment by vivek-insight** [comment]
@sunyao-ms  Thank you for your response. In reference to your reply on 18 September, can u please confirm when this feature/fix will be released to production. Also please refer to our comments on 28 October.

**Comment by sunyao-ms** [comment]
Hi, @vivek-insight , First, we need to confirm whether this is the known issue that we are currently fixing. Please follow these steps to help us diagnose the problem:

1. **Close all other Excel workbooks**: Open a new workbook, open the Excel function arguments window, add a newly created formula, click the cell containing the formula, and then click the fx button to open the function arguments. Check if the issue can be reproduced.

2. **Open another workbook**: Ensure this workbook is the **second workbook** opened on your desktop. Open the Excel function arguments window, add a newly created formula, click the cell containing the formula, and then click the fx button to open the function arguments. Check if the issue can be reproduced.

**Comment by vivek-insight** [comment]
Hi, @sunyao-ms  we have checked the formula through above mention points

1. **Close all other Excel workbooks:** working fine with single excel instance
2. **Open another workbook:** In a two-workbook scenario, the functionality works correctly only for the first Excel instance. However, for the second instance, it shows a #VALUE! error. If the first instance is closed and the **fx** button is clicked in the second instance, the values then display correctly.

Please let us know when the fix is available for the above scenario.

**Comment by sunyao-ms** [comment]
@vivek-insight , thanks for the confirmation. This is the known issue what we are fixing on.
cc: @XuanZhouMSFT

**Comment by vivek-insight** [comment]
@sunyao-ms any update on the fix ?

**Comment by sunyao-ms** [comment]
Yes, our fix has been checked in and we need to wait for the next build version to pick the fix. I will let you know once there is new build version.

**Comment by sunyao-ms** [comment]
Hi, @vivek-insight, the fix has been deployed at insider and reach to production at about early Feb. Please update Excel and make sure your build version is >= 16.0.18502.20000. Thanks.

**Comment by vivek-insight** [comment]
@sunyao-ms Thanks for the reply. I think we need to wait for the actual release to check the fix. Currently latest excel version we have is Microsoft® Excel® for Microsoft 365 MSO (Version 2412 Build 16.0.18324.20092) 64-bit. Please let us know when it fully released to production

**Comment by vivek-insight** [comment]
Hi, @sunyao-ms @XuanZhouMSFT any update on this?  Since February is almost ending, can you provide an estimated timeline for when we can expect this in production?

**Comment by sunyao-ms** [comment]
@vivek-insight , please update your excel and make sure the build version is >= 16.0.18429.20132 to pick up the change.
